README.md

```diff
@@ -22,6 +22,20 @@ then
 python -m jupyter notebook
 ```
 
+Windows Users can quickly set up by using the following steps.
+1st. you need a compiler to run bash commands. You can use the Git Bash for this purpose.
+2nd. The following steps are tested for Python 3.11,3.10,3.9, and 3.8. Installation was successful for any python version except the Python 3.11.
```

Comment:
```suggestion
Windows users should use [Git Bash](https://gitforwindows.org/) and Python version > 3.8 and < 3.11.
```
Just a suggestion to condense the information a bit.

Thanks for testing on all those versions of Python! Do you happen to know what's causing v3.11 to fail?

Reply:
@frankharkins I made the changes you asked and made a new pull request with changes. I think with Python v3.11 cannot install the cmake and after that it gives the error for pyscf and tweedledum. Do you want me to upload a photo of the error itself?